transport/transport-runtime/src/androidTest/java/com/google/android/datatransport/runtime/scheduling/persistence/SpyEventStoreModule.java

```diff
@@ -39,8 +40,9 @@ static SQLiteEventStore sqliteEventStore(
       @WallTime Clock wallClock,
       @Monotonic Clock clock,
       EventStoreConfig config,
-      SchemaManager schemaManager) {
-    return spy(new SQLiteEventStore(wallClock, clock, config, schemaManager));
+      SchemaManager schemaManager,
+      @Named("PACKAGE_NAME") String packageName) {
```

Comment:
Relatedly, can we inject a `Lazy<String>` instead of the string directly? We don't want to query the package name at startup, but rather only do so when it's needed